jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  preset: 'ts-jest/presets/js-with-babel',
+  preset: 'ts-jest/presets/js-with-ts',
```

Comment:
Change presets since we're using TS for everything.

(https://kulshekhar.github.io/ts-jest/docs/getting-started/presets#the-presets)